### PR TITLE
Feat/provider selection component

### PR DESCRIPTION
**Pull request by arnevandoorslaer**

### CODE REVIEW

File: packages/ui-transfer-components/lib/components/provider-list.component.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import { Theme } from '@digita-ai/ui-transfer-theme';
+import { css, CSSResultArray, html, internalProperty, LitElement, property, TemplateResult, unsafeCSS } from 'lit-element';
+import { Source } from '../models/source.model';
+import { ProviderListItemComponent } from './provider-list-item.component';
+
+export class ProviderListComponent extends LitElement {
+
+  @property({ type: Array })
+  public providers: Source[];
```
Comment (woutermont) [inline packages/ui-transfer-components/lib/components/provider-list.component.ts:9]: We want to make a hard distinction between sources (data providers) and issuers (identity providers), so better make an `Issuer` interface with just a `description`field, a `uri` field and an `icon` field.

File: packages/ui-transfer-components/lib/components/provider-list.component.ts [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+  };
+
+  render(): TemplateResult {
+
+    return html`
+      <h1>Select an identity provider to log in</h1>
+      <div class="providers">
+      ${ this.providers.map((provider) => html`
+        <provider-list-item .icon="${provider.icon}" .description="${provider.description}" ?buttonEnabled=${this.buttonsEnabled}></provider-list-item>
```
Comment (woutermont) [inline packages/ui-transfer-components/lib/components/provider-list.component.ts:35]: Here we should listen on the `button-clicked` event of the item. For this, we will also need to change the `onSourceSelected` function; take a look at the one in the consent component of the home component branch (which is a higher order function: it takes a request and returns an event listener function).

File: packages/ui-transfer-components/lib/components/provider-list.component.ts [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+  constructor() {
+
+    super();
+
+    customElements.define('provider-list-item', ProviderListItemComponent);
+
+  }
+
+  onSourceSelected = (e: CustomEvent & { detail: { source: Source } }): void => {
```
Comment (woutermont) [inline packages/ui-transfer-components/lib/components/provider-list.component.ts:22]: Here also switch to Issuers instead of Sources.

**Review by woutermont — CHANGES_REQUESTED** [review]

File: packages/ui-transfer-components/lib/components/provider-list-item.component.ts [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+  }
+  static get styles(): CSSResult[] {
+
+    return [
+      unsafeCSS(Theme),
+      css`
+      .card-container {
+        display: flex;
+        align-items: center;
+        background: #FFFFFF;
```
Comment (lem-onade) [inline packages/ui-transfer-components/lib/components/provider-list-item.component.ts:41]:
```suggestion
        background: var(--colors-background-light);
```

File: packages/ui-transfer-components/lib/components/provider-list.component.ts [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+import { Issuer } from '../models/issuer.model';
+import { ProviderListItemComponent } from './provider-list-item.component';
+
+export class ProviderListComponent extends LitElement {
+
+  @property({ type: Array })
+  public providers: Issuer[];
+
+  @internalProperty()
+  public buttonsEnabled = true;
```
Comment (lem-onade) [inline packages/ui-transfer-components/lib/components/provider-list.component.ts:12]: Don't think you need to specify `public` here, but doesn't really matter. (also in other components)

**Review by lem-onade — APPROVED** [review]

**Pull request merged**